data/remnant/remnant 2 side missions.txt

```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+				`	"I think that is all for now."`
+					goto celebrationrequest
+				`	"That was awe-inspiring."`
+					goto awestruck
+				`	"Why were some of the people crying?`
+					goto askcrying
+				`	"Who leads these ceremonies?"`
+			label ceremonyleader
+			`	Taely gestures take on a tone of conviction. "For something like this, or anything of a social or spiritual nature for that matter, we determine who takes the lead for various parts at the moment based on who is present. Whether we are talking about our connection with society, or our connection to the Universe, we must each forge our own understanding of it. As such, when we gather together for ceremonies such as this, virtually anyone could be chosen to take the lead for one of the segments.`
+			`	"As a result, each celebration takes on a different feel - after all, each of us has our own unique perspective, and bring our own experiences to contribute." She pauses, and adopts a respectful tone. "The ceremony is not just about major events, either. Everyone who takes part brings their own life experiences and memories to it. Memories of friends and family who are no longer with us are common."`
+			choice
+				`	"I think that is all for now."`
+					goto celebrationrequest
+				`	"That was awe-inspiring."`
+					goto awestruck
+				`	"Why were some of the people crying?`
+					goto askcrying
+				`	"That seemed both well-rehearsed and very disorganized."`
+					goto celebrationrehearsed
+				`	"You said 'virtually.' Who is omitted?"`
+			`	Taely scowls a bit at this. "Only those of guilty of the most serious crimes cannot stand as a lead." She pauses, and continues with an amused tone. "Even children take the lead now and then, usually with a parent or sibling to guide them." She smiles at that, then gestures for you to continue asking your questions.`
+			choice
+				`	"I think that is all for now."`
+					goto celebrationrequest
+				`	"That was awe-inspiring."`
+					goto awestruck
+				`	"Why were some of the people crying?`
+					goto askcrying
+				`	"That seemed both well-rehearsed and very disorganized."`
+					goto celebrationrehearsed
+				`	"What constitutes your most serious crimes?"`
+			`	Taely gestures harshly. "Crimes against society, of course. Acting against the best interest of our collective wellbeing is the single most destructive and senseless act someone can do."`
+			choice
+				`	"I think that is all for now."`
+					goto celebrationrequest
+				`	"That was awe-inspiring."`
+					goto awestruck
+				`	"Why were some of the people crying?`
+				`	"That seemed both well-rehearsed and very disorganized."`
+					goto celebrationrehearsed
+			label askcrying
+			`	Taely gestures gently. "Those were friends of the crew of a ship that was destroyed while studying a black hole. They acquired ground-breaking data from it, but perished in the process. Their friends and families are proud of them, of course, but that does not change the fact that they are lost to us mortals."`
```

Comment:
Is this the *only* reason? This whole seemed like a pretty emotional experience. I'd imagine some people would cry of joy as well, and both crying of joy and sorrow could be for a variety of reasons.

Reply:
There is, in all likelyhood, a variety of reasons. It is a fairly emotional thing. I didn't really want to expound at length on all the many different ways there might be someone crying.